go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/k8snetworkplumbingwg/kubemacpool
 
-go 1.22.0
+go 1.22.11
```

Comment:
usually it is better to keep go 1.22.0
and add a new line toolchain 1.22.11

according AI
The go version in go.mod (go 1.2y.0) just sets the minimum required Go version for module compatibility.
The toolchain version (go1.2y.z) is what actually runs your code, so it's the one that determines if you are affected by a CVE.

Reply:
[Fixed](https://github.com/k8snetworkplumbingwg/kubemacpool/compare/d701bff381fefc0f3acdc3941d6603708519a372..1270e2085782eccff9c2ad342c597b4d26273e8c)